app/controllers/application_controller.rb

```diff
@@ -44,16 +45,29 @@ def any_bundle
     bundle
   end
 
+  def check_remaining_disk_space
+    Vmstat.snapshot.disks.each do |disk|
+      disk_percentage = 100 - ((disk.available_blocks / disk.total_blocks.to_f) * 100).round
+      disk_full_msg = "Your disk is #{disk_percentage}% full, please check your remaining hard disk space and " \
+                      'clean up old tests in order to ensure continued stable operation of the Cypress application.'
+      if disk_percentage >= 95
+        flash.now[:disk_space_danger] = disk_full_msg
+      elsif disk_percentage >= 80
```

Comment:
It will take a bit more work to get it to show up once per session. I could perhaps only show it after the sign in route is hit, or I would also be fine with just making the limit 90%